client/src/components/WeatherCard.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import Typography from "@mui/material/Typography";
+import Box from "@mui/material/Box";
+
+const WeatherCard = ({ weatherData }) => {
+  return (
+    <Box
+      sx={{
+        display: "flex",
+        justifyContent: "center",
+        flexDirection: "column",
+        alignItems: "center",
+        margin: "20px",
+        padding: "20px",
+        color: "#37392E",
+      }}
+    >
+      <Typography variant="h3">
+        <span>{weatherData.city}</span>
+      </Typography>
+      <img
+        src={`http://openweathermap.org/img/wn/${weatherData.icon}.png`}
+        alt="Weather Icon"
+      />
```

Comment:
This is totally fine but just wanted to make you aware that you can also use a `Box` for the `img` like this:

`<Box component="img" src={...} alt="..." />`

And then the `<Box>` becomes an `<img>`

Reply:
I will try that next time! Don't want to mess with my `img` too much here because that gradient background color is super finnicky.